src/components/molecules/TableFooter/TableFooter.tsx

```diff
@@ -14,14 +16,11 @@ const TableFooter: React.FC<TableFooterProps> = ({
       <div className="action">
         <div className="focus">Focus on:</div>
         <div className="focus-option">Jazz</div>
-        <label className="switch">
-          <input
-            type="checkbox"
-            checked={!isVideoFocused}
-            onChange={() => setIsVideoFocused(!isVideoFocused)}
-          />
-          <span className="slider" />
-        </label>
+        <Toggler
+          id="showRadio"
```

Comment:
```suggestion
```

Reply:
removed `id` in a64982d7f2094afc0571f671312f576d4007dbcb